utils/general_config.json

```diff
@@ -0,0 +1,219 @@
+
+//generalConfig components
+
+{
+"debeziumConfig": { 
+ "debezium-mysql" : "{{database.hostname}}",
+ "databaseHostname" : "{{database.hostname}}",
+ "databaseServerId" : "{{database.serverId}}",
+ "databasePort" : "{{database.port}}",
+ "databaseUser" : "{{database.user}}",
+ "databasePassword" : "{{database.password}}",
+ "databaseServerName" : "{{database.dbname}}",
+ "databaseWhitelist" : "{{database.schema}}",
+ "offsetStorage" : "org.apache.kafka.connect.storage.FileOffsetBackingStore",
+ "offsetStorageFileName" : "{{database.offsetStorage}}",
+ "databaseHistoryFileFilename" : "{{database.databaseHistory}}"
```

Comment:
ya , @gitcliff ,these settings are no londer used anywhere in the code

Reply:
sure @mozzy11  i have switched the ones you introduced recently in the `DebeziumArgs.class`